### PR TITLE
Enhancement/9332 siwg module

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue:

- #9332 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Modules/Sign_In_With_Google.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+			'order'       => 10,
+			'homepage'    => __( 'https://developers.google.com/identity/gsi/web/guides/overview', 'google-site-kit' ),
+		);
+	}
+
+	/**
+	 * Sets up the module's assets to register.
+	 *
+	 * @since 1.122.0
+	 * @since 1.126.0 Added PAX assets.
```
Comment (zutigrm) [inline includes/Modules/Sign_In_With_Google.php:77]: Seems this was missed, should be `n.e.x.t`

**Review by zutigrm — CHANGES_REQUESTED** [review]
Thanks @eugene-manuilov Looks good, I just left one tiny comment

File: includes/Modules/Sign_In_With_Google.php (hunk 1 above)
Comment (eugene-manuilov) [reply includes/Modules/Sign_In_With_Google.php]: yeah, good catch. Thanks, fixed.

**Review by zutigrm — APPROVED** [review]
Thanks @eugene-manuilov, nice one. LGTM

**Review by tofumatt — APPROVED** [review]
Looks good, just one tiny string tweak that I'll commit, then merge 👍🏻

**Pull request merged**